packages/web-app-files/src/helpers/share/role.ts

```diff
@@ -193,6 +207,41 @@ export const linkRoleUploaderFolder = new LinkShareRole(
   $gettext('uploader'),
   [SharePermissions.create]
 )
+export const spaceViewer = new SpaceShareRole(
+  'spaceViewer',
+  false,
+  $gettext('Viewer'),
+  $gettext('viewer'),
+  [SharePermissions.read]
+)
+export const spaceEditor = new SpaceShareRole(
+  'spaceEditor',
```

Comment:
I think it's still possible to call it `editor`, right? The `space` prefix should not be needed.